.github/workflows/test.yml

```diff
@@ -17,12 +17,14 @@ jobs:
           java-version: ${{ matrix.java }}
           architecture: x64
       - uses: actions/checkout@v3
-    
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Used `${{ github.event.pull_request.head.sha }} ` according to the suggestion on [Push Checkstyle Report](https://github.com/marketplace/actions/push-checkstyle-report)
![Screenshot 2023-07-04 at 9 39 06](https://github.com/line/thrift-gradle-plugin/assets/537312/d776cad8-5016-4f06-81aa-024c638532c2)